src/App.jsx

```diff
@@ -1,104 +1,21 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import { PostList } from './components/PostList';
+
+function findUserByPost(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const postsWithUser = postsFromServer.map(post => ({
+  ...post,
+  user: findUserByPost(post.userId),
+}));
```

Comment:
```suggestion
export const postsWithUser = postsFromServer.map(post => ({
  ...post,
  user: findUserByPost(post.userId),
  comments: commentsFromServer.filter(...)
}));
```

(but better to create some function with that logic as findUserByPost)